database/seeders/TransactionSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\PaymentType;
+use App\Models\Transaction;
+use Illuminate\Database\Seeder;
+
+class TransactionSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Transaction::factory()->count(300)->create();
```

Comment:
What will be the payment type here? Isn't it a necessary field? Otherwise it could give us the null payers.

Reply:
The payment type is selected randomly by the randomly selected checkout's possible payment types (See the Payment Type's `forCheckout` function.)
But I have an idea for the null values, I will check it tomorrow. If I cannot solve it now, I still want to merge this.

Reply:
Okay, I'll also take a look if you cannot find it.

Reply:
The payers here are not approved users. I will fix it.

Reply:
I had to write the filter explicitly, I don't know why the GlobalScope does not work here.
It is fine by me now, however, using `User::Collegists()->random()` should be the optimal solution. 
@luksan47 could you look at it? Or we can just create a new issue.

Reply:
Hm, that seems weird. I could not google the problem.
Let's open an issue and assign it to me, so we can proceed with the merge.